mapie/tests/test_calibration.py

```diff
@@ -157,13 +157,30 @@ def test_other_methods() -> None:
         mapie_cal.fit(X, y)
 
 
-def test_prefit() -> None:
+def test_prefit_cv_argument() -> None:
     """Test that prefit method works"""
     est = RandomForestClassifier().fit(X, y)
     mapie_cal = MapieCalibrator(estimator=est, cv="prefit")
     mapie_cal.fit(X, y)
 
 
```

Comment:
You should also test results with `cv=split`

Reply:
It is tested just after this test. I propose two tests because we nedd to prefit a model with "prefit" and it is not the case with "split".

Reply:
yes tested that the model can be fitted, you didn't tested the results of the calibration, see `test_correct_results` test

Reply:
Ok, I haven't understand that. Okay to add result test with `cv=split`

Reply:
Done technically.

Reply:
Done.